pkg/kubernetes/pod/v1alpha1/kubernetes.go

```diff
@@ -201,3 +264,68 @@ func (k *KubeClient) GetRaw(name string, opts metav1.GetOptions) ([]byte, error)
 	}
 	return json.Marshal(p)
 }
+
+// ExecOutput struct contains stdout and stderr
+type ExecOutput struct {
```

Comment:
Move this struct below the KubeClient struct

Reply:
Done